tests/r/FormatRBearTest.py

```diff
@@ -129,3 +140,9 @@
     valid_files=(good_file_6,),
     invalid_files=(bad_file_3,),
     settings={'r_max_expression_length': '25'})
+
+FormatRBearDefaultSettingsTest = verify_local_bear(
+    FormatRBear,
+    valid_files=(good_file_7,),
```

Comment:
How about adding the valid files check with good_file_1? Just to make sure that it works with brace in newline and inline.